presto-product-tests/bin/product-tests-suite-3.sh

```diff
@@ -22,5 +22,12 @@ presto-product-tests-launcher/bin/run-launcher test run \
     -- -g storage_formats,cli,hdfs_impersonation,authorization -x iceberg,"${DISTRO_SKIP_GROUP}" -e "${DISTRO_SKIP_TEST}" \
     || suite_exit_code=1
 
+# run very selective set of tests on singlenode-kerberos-hdfs-impersonation-with-data-protection so we do not add
+# a lot to runtime
```

Comment:
That's a good thinking, but I do not think it needs to be put in a comment. It's seems obvious why you picked few tests. Unless you want to _also_ explain why these particular tests.